### PR TITLE
(refactor) remove helm instructions in the readme

### PR DESCRIPTION
**Pull request by ksatchit**
Signed-off-by: ksatchit <ksatchit@mayadata.io>

**What this PR does / why we need it**:

- Removes the helm-based instructions to install litmus, as the helm charts are deprecated
- Updates the README with latest engine/experiment schema

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,32 +39,51 @@ runner pod and engine monitor service), which are created & managed by it in ord
 The ChaosEngine is the core schema that defines the chaos workflow for a given application. Currently, it defines the following:
 
 - Application Data (namespace, labels, kind)
-- List of Chaos Experiments to be executed
-- Attributes of the experiments, such as, rank/priority 
-- Execution Schedule for the batch run of the experiments
+- ServiceAccount used for execution of the experiment
+- Chaos Experiment to be executed on the application
+- Attributes of the experiments (overrides defaults specified in the experiment CRs)
+- Chaos resource cleanup Option
```
Comment (shubhamchaudhary) [inline README.md]: Can we also add some details about the auxiliaryAppInfo and anotationCheck field?

File: README.md [hunk 2]
```diff
@@ -40,32 +39,51 @@ runner pod and engine monitor service), which are created & managed by it in ord
...
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  jobCleanUpPolicy: 'delete' 
+  experiments:
+    - name: pod-delete
+      spec:
+        components:
+          env:
```
Comment (shubhamchaudhary) [inline README.md]: env field is not present in `1.0.0 release`. Is it okay to use?

File: README.md (hunk 1 above)
Comment (ksatchit) [reply README.md]: considering we are in master, it should be fine. But taking into account users may copy from here, reverted to previous schema and added note to say (as of 1.0.0)

File: README.md (hunk 2 above)
Comment (ksatchit) [reply README.md]: considering we are in master, it should be fine. But taking into account users may copy from here, reverted to previous schema and added note to say (as of 1.0.0)

**Pull request merged**